docker/README.md

```diff
@@ -0,0 +1,147 @@
+# FIWARE IoT Agent for OMA LightWeight Machine2Machine
+
+[![FIWARE IoT Agents](https://nexus.lab.fiware.org/repository/raw/public/badges/chapters/iot-agents.svg)](https://www.fiware.org/developers/catalogue/)
+[![](https://nexus.lab.fiware.org/repository/raw/public/badges/stackoverflow/iot-agents.svg)](https://stackoverflow.com/questions/tagged/fiware+iot)
+
+This Internet of Things Agent is designed to be a bridge between the
+[OMA](https://www.omaspecworks.org/)
+[Lightweight M2M](https://www.omaspecworks.org/what-is-oma-specworks/iot/lightweight-m2m-lwm2m/)
+protocol and the
+[NGSI](https://swagger.lab.fiware.org/?url=https://raw.githubusercontent.com/Fiware/specifications/master/OpenAPI/ngsiv2/ngsiv2-openapi.json)
+interface of a context broker.
+
+It is based on the
+[IoT Agent Node.js Library](https://github.com/telefonicaid/iotagent-node-lib).
+Further general information about the FIWARE IoT Agents framework, its
+architecture and the common interaction model can be found in the library's
+GitHub repository.
+
+This project is part of [FIWARE](https://www.fiware.org/). For more information
+check the FIWARE Catalogue entry for the
+[IoT Agents](https://github.com/Fiware/catalogue/tree/master/iot-agents).
+
+## How to use this image
+
+The IoT Agent must be instantiated and connected to an instance of the [Orion Context Broker](https://fiware-orion.readthedocs.io/en/latest/), a sample `docker-compose` file can be found below.
+
+If the `IOTA_REGISTRY_TYPE=mongodb`, a [MongoDB](https://www.mongodb.com/) database
+instance is also required - the example below assumes that you
+have a `/data` directory in your hosting system in order to hold database
+files - please amend the attached volume to suit your own configuration.
+
+```yml
+version: '3.1'
+
+volumes:
+  mongodb:
+
+services:
+  iot-agent:
+    image: fiware/lightweightm2m-iotagent
+    hostname: iot-agent
+    container_name: fiware-iot-agent
+    depends_on:
+        - mongodb
+    expose:
+        - "4041"
+        - "7896"
+    ports:
+        - "4041:4041"
+        - "7896:7896"
+    environment:
+        - "IOTA_CB_HOST=orion"
+        - "IOTA_CB_PORT=1026"
+        - "IOTA_NORTH_PORT=4041"
+        - "IOTA_REGISTRY_TYPE=mongodb"
+        - "IOTA_MONGO_HOST=mongo-db"
+        - "IOTA_MONGO_PORT=27017"
+        - "IOTA_MONGO_DB=iotagent-lwm2m"
```

Comment:
Since I'm not familiar with the LWM2M IoT Agent, I don't know which additional "standard" parameters should be added here.

The list currently only contains the common IoT Agent Node Lib ones. I assume a PORT and PROTOCOL would be useful.

Reply:
I understand the ones you have included are the basic ones (in fact, common to other agents) so I think is fine. However, let's wait for @dcalvoalonso feedback on this before closing the issue.

Reply:
This IOTA does not expose more environment variables so this list is complete.